libraries/stdlib/samples/test/samples/collections/sort.kt

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright 2010-2019 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package samples.collections
+
+import samples.*
+import java.time.LocalDateTime
+
+@RunWith(Enclosed::class)
+class Sort {
+
+    class Usage {
+
+        @Sample
+        fun sortArray() {
+            val intArray = intArrayOf(4, 3, 2, 1)
+
+            assertPrints(
+                intArray.joinToString(),
+                "4, 3, 2, 1"
+            )
+
+            intArray.sort()
+
+            assertPrints(
+                intArray.joinToString(),
+                "1, 2, 3, 4"
+            )
+        }
+
+        @Sample
+        fun sortMutableList() {
+            val mutableList = mutableListOf(4, 3, 2, 1)
+
+            assertPrints(
+                mutableList.joinToString(),
+                "4, 3, 2, 1"
+            )
+
+            mutableList.sort()
+
+            assertPrints(
+                mutableList.joinToString(),
+                "1, 2, 3, 4"
+            )
+        }
+
+        @Sample
+        fun sortComparable() {
```

Comment:
Better to name `sortArrayOfComparable`, because it's _array_ being sorted, and not a comparable type.